Documentation/Signoffs/battery-management-signoff.md

```diff
@@ -0,0 +1,144 @@
+# Battery Management Signoff
+
+## Functionality
+
+The primary function of the battery management system is to provide a system that can supply adequate current to the robot's systems. This battery must be able to be recharged without the removal of the battery and be able to be used while being charged.
+
+## Constraints
+
+### Derived from Shall Statements
+
+- C5: The robot shall possess a robust, centralized charging system that does not require the removal of the battery and that allows the robot to be used while being charged.
+	+ This constraint is the most important for this subsystem. This constraint requires that the battery management can provide enough power to the system and allow for the robot to be used while the battery is being charged. Also, that the battery can be charged without removal of the battery. The battery should be able to supply enough energy for the robot to have at least a 30-minute battery life before needing to be recharged.
+
+### Derived from Broader Implications
+
+- C17: The robot shall abide by NEC 310.15(B)(16) when sizing any conductors for the project [1].
+	+ This constraint is relevent to this subsystem because it ensures that all of the conductors and connectors used are able to withstand the maximum possible current that may be required by the robot.
+
+## Schematic
+
+![Battery Management Schematic](../Images/Battery_Management/Battery_Management_Schem.png)
+Figure 1. Battery Management Schematic.
```

Comment:
Could you illustrate more on the adapter? Is it a third-party product? Do you need to control it?

Reply:
I should have explained the adapter more in my update.

Reply:
LOL thanks for the explanation. I think I was thinking about adapter as a power electronic that needs to be designed at first. Today I looked at it and I laughed so hard.